services/QuillLMS/app/models/subscription.rb

```diff
@@ -42,7 +42,6 @@ class RenewalNilStripeCustomer < StandardError; end
   has_many :user_subscriptions
   has_many :users, through: :user_subscriptions
   belongs_to :purchaser, class_name: "User"
-  belongs_to :subscription_type
```

Comment:
This change is not germane to the PR, but I discovered it was unused so I'm cleaning it up.